app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -43,6 +43,7 @@ function initializeCommentsPage() {
                   var buttForComment = document.getElementById('button-for-comment-' + reactions[i].reactable_id);
                   if (buttForComment) {
                     buttForComment.classList.add('reacted');
+                    buttForComment.ariaPressed = true;
```

Comment:
shouldn't this be `buttForComment.setAttribute('aria-pressed', 'true')` ?

Reply:
This works the same as `setAttribute` for aria-pressed.

https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaPressed

Reply:
Oooh! TIL. It doesn't work in Firefox though https://caniuse.com/mdn-api_element_ariapressed

Reply:
@rhymes  Oh no, it looked cool. Will have to change to setAttribute then. 😞